### DISCUSSION TITLE
If a local filepath image is updated/replaced, can ui.image load the new one?

### DISCUSSION THREAD

**Discussion by groucho86** [zauberzeug/nicegui#1922] [Ideas / Feature Requests]
### Question

Here's a sample class:

```py
class page():
    def __init__(self):
        self.card = ui.card()
        self.img = 'current.jpg'
        with self.card.classes().style('width: 640px; height: 360px'):
            ui.image(self.img)
 
        ui.button('Replace', on_click=self.update)
 
    def update(self):
        self.card.clear()
        shutil.copy('new.jpg', 'current.jpg')
        with self.card.classes().style('width: 640px; height: 360px'):
            ui.image(self.img)
```

Desipite clearing the card container, ui.image still displays the old (cached?) image. Is there a way to force nicegui/quasar to reload the updated image?

**Comment by phoskee** [comment]
[Take a look](https://discord.com/channels/1089836369431498784/1167115670790623343)

**Comment by falkoschindler** [comment]
Currently you need to use `app.add_static_files` to serve the image yourself. Then you can add something like some random URL parameter to force reloading the image (see #588).

But we should provide a simpler way to force a reload. An `image.force_reload` method could trigger such a URL parameter to be added on the client.

**Reply by falkoschindler** [reply]
I created PR #1977 with such a `force_reload` method.
